e2e/Makefile

```diff
@@ -152,3 +157,74 @@ setup:
+
+	mkdir -p /tmp/topolvm/worker_lvmd;
+	mkdir -p /tmp/topolvm/lvmd_lvmd;
+	truncate --size=20G $(BACKING_STORE)/backing_store_lvmd_1;
+	$(SUDO) losetup -f $(BACKING_STORE)/backing_store_lvmd_1;
+	$(SUDO) vgcreate -y node-lvmd-myvg1 $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_1 | cut -d: -f1);
+	$(SUDO) lvcreate -y -n csi-node-test-block -L 1G node-lvmd-myvg1;
+	$(SUDO) lvcreate -y -n csi-node-test-fs -L 1G node-lvmd-myvg1;
+
+	# Create additional Volume Groups
+	truncate --size=10G $(BACKING_STORE)/backing_store_lvmd_2;
+	$(SUDO) losetup -f $(BACKING_STORE)/backing_store_lvmd_2;
+	$(SUDO) vgcreate -y node-lvmd-myvg2 $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_2 | cut -d: -f1);
+
+	truncate --size=10G $(BACKING_STORE)/backing_store_lvmd_3;
+	$(SUDO) losetup -f $(BACKING_STORE)/backing_store_lvmd_3;
+	$(SUDO) vgcreate -y node-lvmd-myvg3 $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_3 | cut -d: -f1);
+
+.PHONY: daemonset-lvmd/remove-vg
+daemonset-lvmd/remove-vg:
+	$(SUDO) vgremove -ffy node-lvmd-myvg1;
+	$(SUDO) pvremove -ffy $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_1 | cut -d: -f1);
+	$(SUDO) losetup -d $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_1 | cut -d: -f1);
+	rm -f $(BACKING_STORE)/backing_store_lvmd_1;
+
+	$(SUDO) vgremove -ffy node-lvmd-myvg2;
+	$(SUDO) pvremove -ffy $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_2 | cut -d: -f1);
+	$(SUDO) losetup -d $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_2 | cut -d: -f1);
+	rm -f $(BACKING_STORE)/backing_store_lvmd_2;
+
+	$(SUDO) vgremove -ffy node-lvmd-myvg3;
+	$(SUDO) pvremove -ffy $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_3 | cut -d: -f1);
+	$(SUDO) losetup -d $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_3 | cut -d: -f1);
+	rm -f $(BACKING_STORE)/backing_store_lvmd_3;
+
+.PHONY: daemonset-lvmd/setup-minikube
+daemonset-lvmd/setup-minikube:
+	$(SUDO) apt-get update
+	$(SUDO) apt-get install -y conntrack
+	curl -LO https://github.com/kubernetes/minikube/releases/download/$(MINKKUUBE_VERSION)/minikube-linux-amd64
+	chmod a+x minikube-linux-amd64
+	mv minikube-linux-amd64 $(BINDIR)/minikube
+
+.PHONY: daemonset-lvmd/launch-minikube
+daemonset-lvmd/launch-minikube:
+	$(BINDIR)/minikube start \
+		--vm-driver=none \
+		--kubernetes-version=v$(KUBERNETES_VERSION) \
+		--extra-config=kubelet.read-only-port=10255
+
+.PHONY: daemonset-lvmd/delete-minikube
+daemonset-lvmd/delete-minikube:
+	$(BINDIR)/minikube delete
+
+.PHONY: daemonset-lvmd/update-minikube-setting
+daemonset-lvmd/update-minikube-setting: daemonset-lvmd/delete-minikube daemonset-lvmd/launch-minikube
+	$(SUDO) mkdir -p /var/lib/scheduler
+	$(SUDO) cp -f kubeadm/scheduler-policy.cfg /var/lib/scheduler/scheduler-policy.cfg
+	$(SUDO) cp -f kubeadm/scheduler-config-$(TEST_KUBERNETES_SCHEDULER_VERSION).yaml /var/lib/scheduler/scheduler-config.yaml
```

Comment:
It seems that scheduler policy and config are the same config and policy for daemonset in the e2e directory.
Would you explain any reason to copy them?